saleor/static/dashboard-next/configuration/ConfigurationPage.tsx

```diff
@@ -0,0 +1,95 @@
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import { withStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import * as React from "react";
+
+import { IconProps } from "@material-ui/core/Icon";
+import Container from "../components/Container";
+import PageHeader from "../components/PageHeader";
+import i18n from "../i18n";
+
+interface ConfigurationPageProps {
+  menu: Array<{
+    description: string;
+    disabled: boolean;
+    icon: () => React.Component<IconProps>;
```

Comment:
Why take a component instead of an element?

Reply:
Because we set `className` later.

Reply:
Which I think would be unnecessary if `icon = <Folder color="inherit" fontSize="inherit" />`?